test/kubemark/stop-kubemark.sh

```diff
@@ -45,22 +48,27 @@ rm -rf "${RESOURCE_DIRECTORY}/addons" \
     "${RESOURCE_DIRECTORY}/hollow-node.yaml"  &> /dev/null || true
 
 if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
-  export KUBE_ENABLE_APISERVER_INSECURE_PORT=true
+  export KUBE_ENABLE_APISERVER_INSECURE_PORT=${ENABLE_APISERVER_INSECURE_PORT}
   export KUBERNETES_TENANT_PARTITION=true
   for (( tp_num=1; tp_num<=${SCALEOUT_TP_COUNT}; tp_num++ ))
   do
     export TENANT_PARTITION_SEQUENCE=${tp_num}
     delete-kubemark-master
+    rm -rf "${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tp-${tp_num}"
```

Comment:
We should not need to create these intermediate files or duplicate files. I spent a lot of effort cleaning this up. Let's please not add them back.

Reply:
I agree. this is the one I described today and we can get rid of the local xxxxx-tp-{tp_num} and only have xxxx.tp-{tp_num}. note that we need a set of kubeconfig for each TP clusters since currently the proxy cannot handle all the redirection functionalities, e.g. create tenants.

Reply:
fixed.